### PR TITLE
strip candidate detail text of html

### PR DESCRIPTION
**Pull request by mlevoy**
### What github.com/wevote/WebApp/issues does this fix?
https://github.com/wevote/WebApp/issues/3086
### Changes included this pull request?
Added stripHtmlFromString to candidate detail text coming back from ballotpedia.

### CODE REVIEW

**Comment by mlevoy** [comment]
The only location I saw that we were using candidate summary text from ballotpedia was in the CandidateItem.

File: src/js/components/Ballot/CandidateItem.jsx [hunk 1]
```diff
@@ -108,6 +112,7 @@ class CandidateItem extends Component {
       const twitterDescriptionText = twitterDescription && twitterDescription.length ? `${twitterDescription} ` : '';
       const ballotpediaCandidateSummary = candidate.ballotpedia_candidate_summary;
       let ballotpediaCandidateSummaryText = ballotpediaCandidateSummary && ballotpediaCandidateSummary.length ? ballotpediaCandidateSummary : '';
+      ballotpediaCandidateSummaryText = stripHtmlFromString(ballotpediaCandidateSummaryText);
       ballotpediaCandidateSummaryText = ballotpediaCandidateSummaryText.split(/<[^<>]*>/).join(''); // Strip away any HTML tags
```
Comment (mlevoy) [inline src/js/components/Ballot/CandidateItem.jsx:116]: happy to remove the split or leave it in as an extra precaution

Comment (DaleMcGrew) [reply src/js/components/Ballot/CandidateItem.jsx:116]: @mlevoy any modification to remove the HTML can be put in the `stripHtmlFromString` function. Could you more your `split` code into that function, and search for any other uses of `split` like this so we can consolidate it all in that function?

**Comment by DaleMcGrew** [comment]
Looks great @mlevoy! 👍

**Pull request merged**